src/commands/clearmoderations.js

```diff
@@ -30,7 +30,7 @@ command.execute = async (message, args, database, bot) => {
     }
 
     /** @type {ModerationData[]} */
-    const moderations = await database.queryAll("SELECT id, action, created, value, expireTime - created AS duration, reason, moderator FROM moderations WHERE userid = ? AND guildid = ?",[userId,message.guild.id]);
+    const moderations = await database.queryAll("SELECT id FROM moderations WHERE userid = ? AND guildid = ?",[userId,message.guild.id]);
```

Comment:
Revert this because it will conflict with #171

Reply:
I tested it it doesnt conflict. 171 is already has that commit